src/test/ui/privacy/access_levels.rs

```diff
@@ -0,0 +1,50 @@
+#![feature(rustc_attrs)]
+#![allow(unused)]
```

Comment:
Warnings from the `unused` group are allowed by default in UI tests, so this attribute is unnecessary.